source/screens/TaskScreen.js

```diff
@@ -31,96 +32,77 @@ const ListHeading = styled(Heading)`
   margin-bottom: 8px;
 `;
 
-class TaskScreen extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      activeTasks: [],
-    };
-  }
+const TaskScreen = ({ navigation }) => {
+  const [activeCases, setActiveCases] = useState([]);
+  const { user } = useContext(AuthContext);
+  const { setCurrentForm } = useContext(FormContext);
+  const { getCase, setCurrentCase } = useContext(CaseContext);
 
-  componentDidMount() {
-    this.getTasks();
-  }
+  const sortTasksByLastUpdated = list =>
+    list.sort((a, b) => b.attributes.updatedAt - a.attributes.updatedAt);
 
-  sortTasksByDate = list => list.sort((a, b) => new Date(b.created) - new Date(a.created));
-
-  getTasks = async () => {
+  const getTasks = useCallback(async () => {
     try {
-      const tasks = await StorageService.getData(COMPLETED_FORMS_KEY);
-      this.setState({
-        activeTasks: Array.isArray(tasks) && tasks.length ? this.sortTasksByDate(tasks) : [],
-      });
+      const response = await get('/cases', undefined, user.personalNumber);
+      setActiveCases(
+        Array.isArray(response.data.data) && response.data.data
+          ? sortTasksByLastUpdated(response.data.data)
+          : []
+      );
     } catch (error) {
-      console.log('Tasks not found: ', error);
+      console.error(`Get cases Error: ${error}`);
     }
-  };
+  }, [user]);
```

Comment:
A suggestion for this function is to move the get request for cases to the caseContext so we can reuse the functionality. 

The activeCase state could in that case also be set in the context, if we want to use or update ActiveCases somewhere else in the app.